app/controllers/catalog_controller.rb

```diff
@@ -109,6 +109,7 @@ class CatalogController < ApplicationController
     config.add_show_field solr_name('desc_metadata__addinfolink', :stored_searchable, type: :string), :label => 'Additional Information',
```

Comment:
@ckassel Thanks for pushing on them about this. Pretty shortsighted that
they didn't bother coding around their own feature, but whatevs.

If we feel strongly about keeping it, we could add a chore to look into
managing the additional info links at the collection level and having them
propagate into items. I want to think through logic such that many fields
could be managed at either the collection or item level depending on need.

I will take a look through the hydra-works code at some point and see how /
if they address this for PCDM collections:
https://github.com/projecthydra-labs/hydra-works

-Corey

On Fri, Nov 6, 2015 at 9:53 AM, ckassel <notifications@github.com> wrote:

> In app/controllers/catalog_controller.rb
> <https://github.com/NYULibraries/ichabod/pull/186#discussion_r44143579>:
>
> > @@ -109,6 +109,7 @@ class CatalogController < ApplicationController
> >      config.add_show_field solr_name('desc_metadata__addinfolink', :stored_searchable, type: :string), :label => 'Additional Information',
>
> @chrpr <https://github.com/chrpr> I tried fighting the SDR people to keep
> this in since they requested it in the first place... but then they set up
> a new data stream in which these values don't appear, so I gave up. The
> issue is that the people who requested these things aren't here anymore and
> the new people never understood why they were important. I'm OK with the
> removal from Nyucore. Meanwhile, we've been wanting a review of Nyucore
> Attributes for a long time. There is a chore for that, which has bounced
> around from KADD person to KADD person as y'all have been entering and
> exiting. Feel free to take a look. Thanks!
> … <#150dd4995d7ec155_>
> On Fri, Nov 6, 2015 at 9:35 AM, chrpr <notifications@github.com> wrote:
> In app/controllers/catalog_controller.rb <#186 (comment)
> <https://github.com/NYULibraries/ichabod/pull/186#discussion_r44141802>>:
> > @@ -109,6 +109,7 @@ class CatalogController < ApplicationController >
> config.add_show_field solr_name('desc_metadata__addinfolink',
> :stored_searchable, type: :string), :label => 'Additional Information',
> Surprised to see SDR stakeholders wanting the link to LibGuides removed,
> since they were the original requesters of that functionality. Given that
> they no longer want this, should we consider removing the addinfolink and
> addinfotext functionality entirely? It was only used here, and if we need
> it again we probably want to do it differently anyway. It has a bad smell
> to me -- I wrote it, so that plays in -- and I think it adds needlessly to
> technical debt at this point. I suggest we take it out of Nyucore and the
> catalog_controller completely. — Reply to this email directly or view it on
> GitHub <https://github.com/NYULibraries/ichabod/pull/186/files#r44141802>.
>
> -- Carol Kassel Senior Manager, Digital Library Infrastructure NYU Digital
> Library Technology Services cmk@nyu.edu (212) 992-9246 dlib.nyu.edu
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/NYULibraries/ichabod/pull/186/files#r44143579>.
>



-- 
Corey A Harper
Metadata Services Librarian
New York University Libraries
20 Cooper Square, 3rd Floor
New York, NY 10003-7112
212.998.2479
corey.harper@nyu.edu